Uchu.World/Systems/Behaviors/BehaviorTree.cs

```diff
@@ -318,18 +318,7 @@ private static async Task<BehaviorBase> BehaviorFromInfo(BehaviorInfo info)
                 return null;
             }
 
-            var instance = (BehaviorBase) Activator.CreateInstance(behaviorType);
-            if (instance == default)
-            {
-                Logger.Error($"Could not create behaviour of type {behaviorType}.");
-                return null;
-            }
-            
-            instance.BehaviorId = info.BaseBehavior;
-            BehaviorBase.Cache.Add(instance);
-            
-            await instance.BuildAsync();
-            return instance;
+            return await BehaviorBase.BuildBranch(info.BaseBehavior);
```

Comment:
Make sure to test this very well

Reply:
Love the dab btw

Reply:
Yeah I made this change because the EffectID wasn't included in the jetpack behaviour instance in Uchu even though it does exist (https://lu-explorer.web.app/skills/behaviors/3569), because it is the only behaviour node in the tree and not part of a branch. The old code doesn't set EffectId and EffectHandler for the instance in that case. At least that's how I understand it.